### DISCUSSION TITLE
Preventing Mantine DateInput Calendar Overflow Blocking by Modal

### DISCUSSION THREAD

**Discussion by akashaf** [mantinedev/mantine#4775] [Q&A]
I am using the Mantine library in my React application, specifically the DateInput component. However, I'm encountering an issue where the calendar displayed by the DateInput component overflows and gets blocked by a modal that I have on my page. I want to prevent this calendar from being obscured by the modal. How can I achieve this?

I provided sample code for this issue
```
const AkashafSpace = () => {
  const [opened, { open, close }] = useDisclosure(false);
  const [value, setValue] = useState<Date | null>(null);
  return (
    <Container>
      <Modal opened={opened} onClose={close} title='Authentication' centered>
        <div className='h-52'>
          <DateInput
            value={value}
            onChange={setValue}
            label='Date input'
            placeholder='Date input'
            maw={400}
            mx='auto'
          />
        </div>
      </Modal>
      <Group position='center'>
        <Button onClick={open}>Open centered Modal</Button>
      </Group>
    </Container>
  );
};
```

and some screenshot of above code
![image](https://github.com/mantinedev/mantine/assets/17398301/6455d52e-630e-4076-aacf-ee45d6b8bd7e)

**Expected Behavior:**
open dateinput component calendar overflow from modal component.

**Actual Behavior:**
calendar is overflowing but being blocked by the modal.

**Comment by rtivital** [comment]
`<DateInput popoverProps={{ withinPortal: true }} />`
